typescript/packages/plugins/steer-protocol/src/parameters.ts

```diff
@@ -0,0 +1,94 @@
+import { z } from "zod";
+
+/**
+ * Parameters schema for depositing tokens into a Steer Smart Pool
+ */
+export const depositParametersSchema = z.object({
+    // Pool address where liquidity will be deposited
+    poolAddress: z.string().describe("The address of the Steer Smart Pool"),
+
+    // Amount of token0 to deposit
+    amount0Desired: z.string().describe("The desired amount of token0 to deposit"),
```

Comment:
I would specify that these are in base units, there is another tool called `convertToBaseUnits` that the LLM knows when to call.

"The desired amount in base units of token0 to deposit"

Reply:
I'd do this for all amount parameters